File: backend-python/app/modules/system/entity/sys_user_role.py
from sqlalchemy import BigInteger
from sqlalchemy.orm import Mapped, mapped_column

from app.modules.system.entity.base import Base


class SysUserRole(Base):
    __tablename__ = "sys_user_role"

    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
    user_id: Mapped[int] = mapped_column(BigInteger)
    role_id: Mapped[int] = mapped_column(BigInteger)

File: backend-go/internal/modules/system/entity/sys_user_role.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysUserRole struct {
	ID        common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	UserID    common.Int64String `gorm:"column:user_id" json:"userId"`
	RoleID    common.Int64String `gorm:"column:role_id" json:"roleId"`
	CreatedAt *time.Time         `gorm:"column:created_at" json:"createdAt"`
}

func (SysUserRole) TableName() string { return "sys_user_role" }
